### DISCUSSION TITLE
Please do not rewrite "public" history

### DISCUSSION THREAD

**Discussion by polyzen** [marlonrichert/zsh-autocomplete#565] [General]
Earlier today it seems the main branch was rewritten, which ideally should not be done.

A good read: https://www.mail-archive.com/dri-devel@lists.sourceforge.net/msg39091.html

**Comment by marlonrichert** [comment]
That wasn't today, but yes, I did that because I accidentally pushed something I really didn't want to push. I know that it's a bad practice; I do it only in extreme circumstances. I'll try to avoid it next time -- or perhaps just be less picky about the commit history. 🙂 

Were there any problems caused by this?

**Comment by polyzen** [comment]
> That wasn't today, but yes, I did that because I accidentally pushed something I really didn't want to push. I know that it's a bad practice; I do it only in extreme circumstances. I'll try to avoid it next time -- or perhaps just be less picky about the commit history. :slightly_smiling_face:

👍

> Were there any problems caused by this?

Not that I'm aware of.
